blog/2024-05-08-v0-format-modifications.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Navigating Change in Conda-Build Recipe Formats: Balancing Legacy and Innovation"
+slug: "2024-05-08-v0-format-modifications"
+authors: [kenodegard]
```

Comment:
How do we credit ChatGPT here since it's technically not https://github.com/chatgpt?

Reply:
never mind, forgot we have an actual authors file